client/webpack.common.config.js

```diff
@@ -6,12 +6,8 @@ module.exports = {
 
   // the project dir
   context: __dirname,
-  entry: ['./assets/javascripts/App'],
+  entry: ['jquery', 'jquery-ujs', './assets/javascripts/App'],
```

Comment:
That's correct. But it can be easily linked in application.js and squashed by assets pipeline. Moreover if there will be multiple bundles, this needs to be done to avoid duplication of vendors libs in each bundle, so it will be example how to do it with webpack.